src/python_inspector/utils.py

```diff
@@ -54,6 +54,20 @@ def write_output_in_file(output, location):
     return output
 
 
+def write_resolved_packages(package_list, requirements_file):
```

Comment:
@arijitde92 please add a separate unit test for this function, separated from CLI logic

Reply:
@TG1999 , I have written an unit test in test_cli.py (See [here](https://github.com/arijitde92/python-inspector/blob/235a0563699e3d718db874f3819a5f4cc976784f/tests/test_cli.py#L448)). Do I need to write another unit test elsewhere (maybe in test_utils.py?)?
Also could you please clarify what "separated from CLI logic" means?
I assume that you mean to test for a valid `package_list`. In that case do I need to create a dummy `package_list`?

Reply:
Hi @TG1999 , could you please help in clarifying what kind of unit test you need? And in which while should I write the test?

Reply:
@arijitde92 yes, please write a test in `test_utils.py`

Reply:
I have written two unit tests in test_utils.py. Please check.